doc/developer-guide/api/functions/TSAcceptor.en.rst

```diff
@@ -44,8 +44,7 @@ An acceptor thread listens for incoming connections and creates the virtual conn
 
 :func:`TSAcceptorGet` returns :type:`TSAcceptor` object that created :arg:`sslp`.
 
-:func:`TSAcceptorGetbyID` returns the :type:`TSAcceptor` object identified by :arg:`id`. :type:`TSAcceptor` represents the acceptor object created by the core
-traffic server.
+:func:`TSAcceptorGetbyID` returns the :type:`TSAcceptor` object identified by :arg:`id`. :type:`TSAcceptor` represents the acceptor object created by the server.
```

Comment:
Is this a mistake?   What server?

Reply:
What is a core traffic server?

Reply:
Walt's right - that should be "proxy" or "core". "server" will cause people to think it is on a remote box.

Reply:
We discussed this in the PR/issue meeting today, we suggest simply using `|TS|`.

Reply:
Can we just delete the word "core"?  "server" seems too ambiguous.